### PR TITLE
Littlefoot lx dev

### PR DESCRIPTION
**Pull request by tominabox1**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/tominabox1/littlefoot_lx/keymaps/via/via.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
+{
```
Comment (drashna) [inline keyboards/tominabox1/littlefoot_lx/keymaps/via/via.json:1]: via json doesn't belong in this repo, and needs to be removed and pr'ed to the-via/keyboards

File: keyboards/tominabox1/littlefoot_lx/readme.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Littlefoot LX
+
+![littlefoot_lx](https://i.imgur.com/0WhgT5y.jpeg)
```
Comment (drashna) [inline keyboards/tominabox1/littlefoot_lx/readme.md:3]:
```suggestion
![littlefoot_lx](https://i.imgur.com/0WhgT5yh.jpeg)
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/tominabox1/littlefoot_lx/readme.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Littlefoot LX
+
+![littlefoot_lx](https://i.imgur.com/0WhgT5yh.jpeg)
+
+*A short description of the keyboard/project*
```
Comment (drashna) [inline keyboards/tominabox1/littlefoot_lx/readme.md:5]: A short description?

File: keyboards/tominabox1/littlefoot_lx/rev1/rules.mk [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+#
+BOOTMAGIC_ENABLE = yes	    # Enable bootmagic lite
+MOUSEKEY_ENABLE = no	    # Mouse keys
+EXTRAKEY_ENABLE = no	    # Audio control and System control
+CONSOLE_ENABLE = no	        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes		    # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/tominabox1/littlefoot_lx/rev1/rules.mk:18]:
```suggestion
NKRO_ENABLE = yes		    # Enable N-Key Rollover
```

File: keyboards/tominabox1/littlefoot_lx/rev2/rules.mk [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+#
+BOOTMAGIC_ENABLE = yes	    # Enable bootmagic lite
+MOUSEKEY_ENABLE = no	    # Mouse keys
+EXTRAKEY_ENABLE = no	    # Audio control and System control
+CONSOLE_ENABLE = no	        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes		    # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/tominabox1/littlefoot_lx/rev2/rules.mk:18]:
```suggestion
NKRO_ENABLE = yes		    # Enable N-Key Rollover
```

File: keyboards/tominabox1/littlefoot_lx/rev2/config.h [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+/* disable debug print */
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
```
Comment (drashna) [inline keyboards/tominabox1/littlefoot_lx/rev2/config.h:59]:
```suggestion
```

File: keyboards/tominabox1/littlefoot_lx/rev2/rules.mk [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = yes	    # Enable bootmagic lite
+MOUSEKEY_ENABLE = no	    # Mouse keys
+EXTRAKEY_ENABLE = no	    # Audio control and System control
+CONSOLE_ENABLE = no	        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (drashna) [inline keyboards/tominabox1/littlefoot_lx/rev2/rules.mk:17]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/tominabox1/littlefoot_lx/rev2/rules.mk [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+#
+BOOTMAGIC_ENABLE = yes	    # Enable bootmagic lite
+MOUSEKEY_ENABLE = no	    # Mouse keys
+EXTRAKEY_ENABLE = no	    # Audio control and System control
+CONSOLE_ENABLE = no	        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes		    # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no        # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no         # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
```
Comment (fauxpark) [inline keyboards/tominabox1/littlefoot_lx/rev2/rules.mk:18]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = no        # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/tominabox1/littlefoot_lx/readme.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+    Rev1 is the PCB that shipped with the group buy run by RPMorrison on Tach Factory. These PCBs can be identified by the bodge wire on column 4.
+    Rev2 is the PCB available after this group buy.
+
+Make example for this keyboard (after setting up your build environment):
+
+    make littlefoot_lx:default
+
+Flashing example for this keyboard:
+
+    make littlefoot_lx:default:flash
```
Comment (fauxpark) [inline keyboards/tominabox1/littlefoot_lx/readme.md:21]:
```suggestion
    make tominabox1/littlefoot_lx:default

Flashing example for this keyboard:

    make tominabox1/littlefoot_lx:default:flash
```

File: keyboards/tominabox1/littlefoot_lx/readme.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
+# Littlefoot LX
+
+![littlefoot_lx](https://i.imgur.com/0WhgT5yh.jpeg)
+
+The Littlefoot LX is a premium, integrated pcb version of the Littlefoot by The_Royal. 
+
+* Keyboard Maintainer: [tominabox1](https://github.com/tominabox1)
+* Hardware Supported: *Littlefoot LX integrated ATMega32u4*
+* Hardware Availability: *https://tachfactory.bigcartel.com*
```
Comment (fauxpark) [inline keyboards/tominabox1/littlefoot_lx/readme.md:9]:
```suggestion
* Hardware Supported: Littlefoot LX integrated ATMega32u4
* Hardware Availability: https://tachfactory.bigcartel.com
```

File: keyboards/tominabox1/littlefoot_lx/rev1/rules.mk [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+#
+BOOTMAGIC_ENABLE = yes	    # Enable bootmagic lite
+MOUSEKEY_ENABLE = no	    # Mouse keys
+EXTRAKEY_ENABLE = no	    # Audio control and System control
+CONSOLE_ENABLE = no	        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes		    # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no        # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no         # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
```
Comment (fauxpark) [inline keyboards/tominabox1/littlefoot_lx/rev1/rules.mk:18]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = no        # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**